src/Symfony/Component/Routing/RouteCollectionBuilder.php

```diff
@@ -323,6 +324,8 @@ public function build()
     /**
      * Generates a route name based on details of this route.
      *
+     * @param Route $route
```

Comment:
should be removed, it does not add any value over the type-hint in the method argument definition.